### PR TITLE
flake8

### PR DESCRIPTION
**Pull request by mlautman**
updating to flake8

### CODE REVIEW

File: src/rqt_console/console_widget.py [hunk 1]
```diff
@@ -340,7 +388,8 @@ def _add_exclude_filter(self, filter_index=False):
 
         # pack the new filter tuple onto the filter list
         self._exclude_filters.append(
-            (newfilter, FilterWrapperWidget(newwidget, self.filter_factory[filter_index][0]), filter_index))
+            newfilter, FilterWrapperWidget(
```
Comment (brawner) [inline src/rqt_console/console_widget.py]: Is there a tuple missing? Does append automatically create a tuple here?

File: src/rqt_console/console_widget.py [hunk 2]
```diff
@@ -291,10 +333,12 @@ def _add_highlight_filter(self, filter_index=False):
 
         # pack the new filter tuple onto the filter list
         self._highlight_filters.append(
-            (newfilter, FilterWrapperWidget(newwidget, self.filter_factory[filter_index][0]), filter_index))
+            newfilter, FilterWrapperWidget(
```
Comment (brawner) [inline src/rqt_console/console_widget.py]: Tuple parens are missing here

**Review by brawner — CHANGES_REQUESTED** [review]

File: src/rqt_console/console_widget.py (hunk 2 above)
Comment (mlautman) [reply src/rqt_console/console_widget.py]: great catch!

**Comment by mlautman** [comment]
@brawner fixed

**Comment by mlautman** [comment]
Comments addressed. Merging per @brawner's instructions

**Pull request merged**